sdk/Lusid.Sdk.Tests/tutorials/Ibor/PortfolioCashFlows.cs

```diff
@@ -52,7 +53,7 @@ public void ExamplePortfolioCashFlowsForBonds()
                 bond);
             
             // CALL api to get cashflows at maturity
-            var maturity = bond.MaturityDate.Value;
+            var maturity = bond.MaturityDate;
```

Comment:
Why this change?

Reply:
There is no `Value` on that property anymore:

<img width="735" alt="2021-05-21_10-03-45" src="https://user-images.githubusercontent.com/13220/119112324-ded02b80-ba1b-11eb-8ef6-8e703e629b53.png">

Reply:
Makes sense, can't be null anymore!